pkg/apis/machine/v1beta1/machine_types_test.go

```diff
@@ -92,8 +92,8 @@ func TestRoundTripMachine(t *testing.T) {
 		// losing data
 		spec := &MachineSpec{}
 		status := &MachineStatus{}
-		fuzzer.Fuzz(spec)
-		fuzzer.Fuzz(status)
+		machineFuzzer.Fuzz(spec)
+		machineFuzzer.Fuzz(status)
```

Comment:
Unless there was a compelling reason, I would not include cosmetic changes like this

Reply:
I've noticed a go lint warning on these, and since I'm a big advocate of the [Boy Scout rule](https://biratkirat.medium.com/step-8-the-boy-scout-rule-robert-c-martin-uncle-bob-9ac839778385) :-) - I try to follow it myself.
However it is defiantly something I can change back in case there is any objection from cloud-team.

Reply:
IMO let's keep the diff minimal for now. I appreciate boy scout rules, but, this PR is big enough as it is ;)